electrum/commands.py

```diff
@@ -1302,6 +1333,7 @@ def eval_bool(x: str) -> bool:
     'iknowwhatimdoing': (None, "Acknowledge that I understand the full implications of what I am about to do"),
     'gossip':      (None, "Apply command to gossip node instead of wallet"),
     'connection_string':      (None, "Lightning network node ID or network address"),
+    'strategies': (None, "Select RBF any one or multiple RBF strategies in any order, separated by ','; Options : 'CoinChooser','DecreaseChange','DecreasePayment' "),
```

Comment:
please also add an entry here for `new_fee_rate`, so that the command has a useful help text (you can see this if you run `./run_electrum help bumpfee`).
see the entry for the similar `feerate` param

Reply:
Done